website/docs/proxy/index.md

```diff
@@ -6,12 +6,365 @@ tags: [podman-desktop, installing, windows, macos, linux]
+- However, it does not contain additional utilities such as Compose or Kind.
+
+1. Download the Disk Image for restricted environments.
+
+   From a host that has access to external resources, go to the [macOS downloads page](/downloads/macos), and click the `Disk Image for restricted environments` link.
+
+1. Copy the downloaded file to the macOS host in a restricted environment.
+
+1. Double-click the copied the file.
+
+1. Drag the Podman Desktop icon to the Applications folder.
+
+1. The **Dashboard** screen displays: _<Icon icon="fa-solid fa-info" size="lg" /> Podman needs to be set up_.
+
+   ![Podman needs set up screen](img/dashboard-podman-needs-set-up.png)
+
+   Click the **Set up** button.
+
+   Review and validate all confirmation screens to set up the Podman Machine.
+
+   :::tip
+
+   To route the network traffic through your VPN, in the **Create Podman machine** screen, enable the **User mode networking (traffic relayed by a user process)** option.
+
+   :::
+
+</TabItem>
+<TabItem value="linux" label="Linux">
+
+To install the Podman Desktop and Podman in a restricted environment, consider using the Podman Desktop archive for restricted environments.
+
+- It has all artifacts required to install Podman Desktop.
+- It does not require Internet access to download resources during installation.
+- It does not contain additional utilities such as Podman CLI, Compose or Kind.
+
+1. The Podman Destkop archive for restricted environments does not contain Podman CLI.
+
+   To install Podman, go to [the Podman website](https://podman.io/), and follow the installation instructions.
+
+1. To download the Podman Desktop archive, from a host that has access to external resources, go to [the Linux Downloads page](https://podman-desktop.io/downloads/linux), and click the **AMD64 binary (tar.gz)** link.
+
+1. Copy the downloaded file to the Linux host in a restricted environment.
+
+1. Extract the archive content.
+
+1. In the extracted directory, open the `podman-desktop` executable file.
+
+</TabItem>
+</Tabs>
+
+#### Verification
+
+- In the **Dashboard**, the **Podman** tile displays _Podman is running_.
+
+  ![Podman is running screen](img/dashboard-podman-is-running.png)
+
+## Using a proxy
+
+You need:
+
+- `<your.proxy.tld:port>`: Your proxy URL.
+- Optionally: your proxy Certificate Authorities (CA) in Privacy-Enhanced Mail (PEM) format.
+
+<Tabs groupId="operating-systems">
+<TabItem value="win" label="Windows">
+
+1. To enable proxy settings, go to **Settings > Proxy**, toggle on **Proxy configuration enabled**, set your proxy URL, and validate.
+
+   ![Proxy settings](img/proxy-settings.png)
+
+   <details>
+   <summary>
+
+   Optionally, to use a proxy requiring a custom Certificate Authorities:
+
+   </summary>
+   <div>
+
+   1. Store your proxy Certificate Authorities (CA) in Privacy-Enhanced Mail (PEM) format, in your home directory, in the `proxy_ca.pem` file.
```

Comment:
Bit confused on this part and may require better explanation? I don't really understand the "in your home directory in the file".

Like... My Documents? C:/Users/admin/ ? is it in the home dir?

Reply:
```suggestion
   1. Store your proxy Certificate Authorities (CA), in Privacy-Enhanced Mail (PEM) format, in the `proxy_ca.pem` file.
```